### DISCUSSION TITLE
Environment Variable for UPDATER_PERIOD syntax change?

### DISCUSSION THREAD

**Discussion by yeahmyusernameis32characterslong** [qdm12/gluetun#2189] [Q&A]
The environment Variable for UPDATER_PERIOD no longer recognises the H, for example 24H so the container hosting the application shutdown. For now I have commented out the variable. 

Can anyone please advise what the new syntax would be?

**Comment by ezekieldas** [comment]
I'm using `- UPDATER_PERIOD=1h` (note lower case)

**Reply by yeahmyusernameis32characterslong** [reply]
I'll give the lowercase a try, thank you
